src/main/scala/dx/core/io/DxWorkerPaths.scala

```diff
@@ -15,8 +41,59 @@ import wdlTools.util.Logger
   * files created with stdlib calls like "write_lines".
   * @param rootDir the root directory - typically the user's home directory
   */
-case class DxWorkerPaths(rootDir: Path)
-    extends ExecPaths(rootDir, rootDir.resolve(DxWorkerPaths.TempDir)) {
+case class DxWorkerPaths(rootDir: Path) extends BaseEvalPaths with ExecPaths {
+  def getRootDir(ensureExists: Boolean = false): Path = {
+    getOrCreateDir("root", rootDir, ensureExists)
```

Comment:
I cannot find where `getOrCreateDir` is defined. If rootDir is /home/dnanexus, is `getRootDir()` it going to get/create a directory at /home/dnanexus/root?

Reply:
It is in dxCommon. "root" is the key used for caching.

Reply:
`import dx.util.{BaseEvalPaths, ExecPaths, Logger}`

Reply:
https://github.com/dnanexus/dxScala/blob/main/common/src/main/scala/dx/util/Paths.scala